.changeset/tidy-eggs-try.md

```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": major
```

Comment:
If we already do this, the next release will be a major. Ist das okay?

Reply:
Oh yes sure. I thought just about that is a breaking change.

Reply:
Until we make a 1.0 it's fine to do a minor. We did this until now 🤠